ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageVulnerabilities.tsx

```diff
@@ -148,7 +151,10 @@ function DeploymentPageVulnerabilities({
     const hiddenSeverities = getHiddenSeverities(querySearchFilter);
     const hiddenStatuses = getHiddenStatuses(querySearchFilter);
 
-    const query = getVulnStateScopedQueryString(querySearchFilter, currentVulnerabilityState);
+    const query = getVulnStateScopedQueryString(
+        { ...baseSearchFilter, ...querySearchFilter },
```

Comment:
As you know, my mind infers patterns. Here is what I found:
1. Double spread:
    * `...baseSearchFilter` precedes `...querySearchFilter` here and in total of 2
    * `...baseSearchFilter` follows `...querySearchFilter` in total of 3 which seems more intuitive?
2. `...baseSearchFilter` precedes property that has literal key, which you can see does not override it

Reply:
Although functionality it doesn't make a difference (_yet_), I think it makes sense for the `baseSearchFilter` to be the first object spread in. Although it depends on what the intended future context is. Is the base filter the user's "starting point" that can be overridden? Or is it the immutable default that the user should not be able to change?

In this case, the options being passed to the `baseSearchFilter` and what the user can apply manually are disjoint, so it is hard to make a decision yet. I'll move these assignments so that the object is spread in first though, just for the sake of consistency now.

Reply:
Err actually lets do this ordering:
- immutable defaults (e.g. A deployment page's ID)
- baseSearchFilter
- user applies filters

Reply:
Super, can you put your reasoning on the team meeting agenda (even if for next year) for awareness?